### PR TITLE
Implement forward ECal insert

### PR DESCRIPTION
**Pull request by rymilton**
### Briefly, what does this PR introduce?
Adds the forward ecal insert detector. Depends on PR #145.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [x] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No changes needed so far.
### Does this PR change default behavior?
Yes, the ecal insert is added to the default simulation.

### CODE REVIEW

**Comment by rymilton** [comment]
@wdconinc The geometry for this PR is done, but I'm not sure how to configure the output. This ecal insert implementation uses a homogeneous fiber structure, so it needs to be treated like the homogeneous forward ecal.

**Comment by rymilton** [comment]
But I'm assuming this is done in EICrecon and not in epic?

File: compact/definitions.xml [hunk 1]
```diff
@@ -169,12 +169,13 @@ The unused IDs below are saved for future use.
       Unused IDs: 106-109
```
Comment (wdconinc) [inline compact/definitions.xml:169]:
```suggestion
      Unused IDs: 107-109
```

**Review by wdconinc — APPROVED** [review]

**Comment by wdconinc** [comment]
Yes, reconstruction would be done by EICrecon.

**Comment by wdconinc** [comment]
I approved. Most(all?) suggested changes are trivial, can be just applied in the browser by batching or clicking individually. I'll reapprove when applied. I'll turn on auto-merge.

**Review by wdconinc — APPROVED** [review]

**Pull request merged**